src/prefect/events/schemas.py

```diff
@@ -286,30 +409,106 @@ class ExistingAutomation(Automation):
     id: UUID = Field(..., description="The ID of this automation")
 
 
-@copy_model_fields
```

Comment:
Since our `@copy_models_fields` stuff won't be compatible with pydantic v2, I decided to go ahead and drop them from this module.